docs/resources/vault_radar_source_github_enterprise.md

```diff
@@ -0,0 +1,45 @@
+---
+page_title: "hcp_vault_radar_source_github_enterprise Resource - terraform-provider-hcp"
+subcategory: ""
```

Comment:
@jasonpilz I don't want the documentation to show up in the online docs, so I left `subcategory` blank.